python/HeaderService/hslib_salobj.py

```diff
@@ -671,11 +683,25 @@ async def announce(self, imageName):
 
         await self.evt_largeFileObjectAvailable.set_write(**kw)
         self.log.info(f"Sent {self.config.hs_name} largeFileObjectAvailable: {kw}")
+        self.completed_OK[imageName] = True
+
+    async def upload_to_s3(self, fileobj, key, imageName, nt=2):
```

Comment:
This is fairly self-explanatory, but a minimal docstring would still be welcome.

Reply:
Will do.